src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +47,9 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream()
+                .filter(n -> n > 3)
+                .map(n -> n * 2)
+                .reduce(0, Integer::sum);
```

Comment:
```suggestion
            .mapToInt(n -> n * 2)
            .sum();
```

요렇게하면, `0`에 대한 개념을 생략가능할것 같아요!